packages/components/src/local/atoms/zip-dropzone/README.md

```diff
@@ -0,0 +1,27 @@
+# Planned Route documentation
+
+This is an example readme file for the 'zip-dropzone' component.
+
+## UI Example
+
+<!-- STORY -->
+
+## Code example
+
+<!-- SOURCE -->
+
+## Props
+
+<!-- PROPS -->
+
+## Usage
+
+### ES6 Import
+```js
+import ImageDropzone from './zip-dropzone'
+```
+
+### CommonJS
+
+```js
+const ImageDropzone = require('./zip-dropzone')
```

Comment:
A couple of copy/paste issues with this file.  It's not for `Planned Route` (line 1), and it's not for `ImageDropzone` (lines 21 and 27).